src/services/scriptingservice.h

```diff
@@ -172,6 +172,12 @@ class ScriptingService : public QObject {
     Q_INVOKABLE QVariant getApplicationSettingsVariable(
         const QString &key, const QVariant &defaultValue = QVariant());
 
+    Q_INVOKABLE QList<int> fetchSubFoldersByParentId(int parentId);
+
+    Q_INVOKABLE QString getSubFoldersName(int id);
```

Comment:
Why is this called `SubFolders`, instead of `SubFolder`?

Reply:
Typo